src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -39,8 +42,7 @@ export class SendVerifyComponent {
 
   private onSuccess() {
     this.sendButton.setSuccess();
```

Comment:
In addition to this, it is necessary to add `this.sendButton.setDisabled();`

Reply:
Fixed but we don't understand why it should be added. The disabled state was set before injecting transaction and not changed. 'setSuccess()' is not changed the disabled state as well.

Reply:
It is necessary because the disabled state is set before injecting the transaction to `backButton`, not to `sendButton`. Without the change, after sending the transaction the result is this:

![enabled](https://user-images.githubusercontent.com/34079003/40556996-a70a024a-601c-11e8-9845-4fc221cb32ac.png)

In that state the user could press the button again. With the change the result is this:

![disabled](https://user-images.githubusercontent.com/34079003/40557035-c84b6e80-601c-11e8-8af6-cf4115dd6da3.png)